ads/how-to-guides/dynamic-backdrops.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 8
+---
+
+# Dynamic backdrops
+
+Some layouts, such as L-shape and Double Box, make use of a backdrop. These can be specified on both the monetised stream and the ad break in 2 ways:
```

Comment:
Nit-pick: currently we have always used `monetized streams` since we usually take the American English spelling. We probably want to keep this consistent.